modules/compute/gke-node-pool/main.tf

```diff
@@ -44,6 +44,7 @@ resource "google_container_node_pool" "node_pool" {
   name           = var.name == null ? var.machine_type : var.name
   cluster        = var.cluster_id
   node_locations = var.zones
+  version        = var.is_gke_sandbox ? var.node_version : null
```

Comment:
Should we expose the `version` argument to the user here and in cluster? I believe it'll be useful to the user

Reply:
I would like to defer this in the interest of moving fast. Will log a ticket to consider this